sormas-backend/src/main/java/de/symeda/sormas/backend/deletionconfiguration/CoreEntityDeletionService.java

```diff
@@ -88,6 +91,17 @@ public void executeAutomaticDeletion() {
 			});
 		});
 
+		deleteOrphanEntities();
```

Comment:
looks ok
what i don't get is why do we need to remove the symptoms here
why not remove them on the specific services like CaseService delete methods ?

Reply:
Good question, asking myself the same one; why not manually delete the symptoms of visits while deleting these visits? I know that the deletion no longer cascades, but wouldn't it make sense to manually take care of it right there?

Reply:
Indeed the symptoms deletion could be added to deletePermanent method of a case. This way we would have to run this query for each case deletion while with the current implementation we run this query only once. Since this is a performance improvement ticket and because this approach is used already for other entities and we might use it in the future for performance improvements I decided to adopt this solution.